src/index.ts

```diff
@@ -8,6 +8,11 @@ export {
     TactConstEvalError,
     TactErrorCollection,
 } from "./errors";
+export {
+    optionsSchema,
+    projectSchema,
+    configSchema,
```

Comment:
While we're at it, we might also export `parseConfig()` and verifyConfig()` here :)

Reply:
Let's do it later, I believe we should create a separate index file